functional/mappings/reponse-test-query-param-mapping.json

```diff
@@ -0,0 +1,13 @@
+{
+  "request": {
+    "urlPattern": "/resource?startDate=2018-02-01&endDate=2018-02-28&product-code=10&product-code=j1j1j1",
```

Comment:
Do we need to include all static values of query parameters here?  What if you changed it to urlPathPattern and include only path?

Reply:
urlPattern supports regex, thus the 'Pattern' part of the field. So something like "urlPattern": "/resource\\?startDate=[0-9]+&endDate=[0-9]+&product-code=[a-z]+&product-code=[a-z]+".

Reply:
That's right!